systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -165,4 +161,40 @@
      * Tag for recovery tests
      */
     String RECOVERY = "recovery";
+    /**
```

Comment:
If you add spaces before the javadoc comments, it would be much more readable.

Reply:
I added new line before each javadoc + updated `TESTING.md`